crates/rethnet_eth/src/remote.rs

```diff
@@ -231,24 +231,70 @@ impl RpcClient {
         Ok(success.result)
     }
 
+    fn hash_string(input: &str) -> String {
+        use std::hash::Hasher;
+        let mut hasher = std::collections::hash_map::DefaultHasher::new();
```

Comment:
The hashbrown hashmap is faster, so I recommend using their hashing algorithm - if it suffices the requirements of your use case